lib/render.js

```diff
@@ -65,9 +75,16 @@ function makeState(data, meta) {
     indices = getIndices(meta._layoutRef || meta._ref, data),
     componentList = indices && indices.components || [];
 
+  // only get component variations if there is a styleguide
+  if (_.has(meta.locals.site, 'styleguide')) {
+    componentVariations = styleguide.getVariations(meta.locals.site.styleguide);
```

Comment:
⚠️ You're reassigning a module-level variable here while handling the rendering of a specific page. This may lead to scenarios in which `componentVariations` is not set as you expect for subsequent page renderings.